Src/UIComponents/GoalTile/GoalTile.ts

```diff
@@ -2,29 +2,36 @@ import createGauge from './Gauges';
 import { createElement } from '../utils/utils';
 
 //elementID - element ID to append the tile to
-function generateGoalTile(name: string, elementID:string, currentValue:number, minValue:number, maxValue:number, unit:string){
+function generateGoalTile(name: string, elementID:string, currentValue:number, minValue:number, maxValue:number, unit:string, isReversed:boolean){
         const goalTile = createElement('div', "goaltile");
         const header = createElement('span', "goalheader", name.toUpperCase());
         const gauge = createElement('div', "goalgauge");
         gauge.id = `${name}gauge`;
 
-        goalTile.append(header,gauge);
+        goalTile.append(header, gauge);
         const parentElement = document.querySelector(`#${elementID}`);
         parentElement.appendChild(goalTile);
-        createGauge(`${name}gauge`, currentValue, minValue, maxValue);
 
+        //create the gauge
+        createGauge(`${name}gauge`, currentValue, minValue, maxValue, isReversed);
         
         //adding an unit to the texts
-        document.querySelectorAll('.goalgauge > svg > text > tspan').forEach(text =>{
+        document.querySelectorAll(`#${elementID} .goalgauge > svg > text > tspan`).forEach(text =>{
                 const unitText = document.createTextNode(` ${unit}`);
                 if(text.innerHTML){
                         text.appendChild(unitText);
                 }               
```

Comment:
I think that it could be replaced by one line
`text.innerHTML += " unit"`

Reply:
done, kept the 'if' as it's required for keeping the percentages without the unit